api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,27 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
-    
+
+	<changeSet id="20181402-TRUNK-5339" author="esirkings">
+		<preConditions onFail="MARK_RAN">
+			<not><indexExists indexName="diagnosisId"/></not>
```

Comment:
Since you are creating a table, the precondition should be about checking if the table already exists.

Reply:
Noted. Refactoring it.